### PR TITLE
3 DSK cards (Omnivorous Flytrap and friends)

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/omnivorous_flytrap.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+T:Mode$ Attacks | ValidCard$ Card.Self | Secondary$ True | Execute$ TrigPutCounter | Delirium$ True | TriggerDescription$ Delirium — Whenever CARDNAME enters or attacks, if there are four or more card types among cards in your graveyard, distribute two +1/+1 counters among one or two target creatures. Then if there are six or more card types among cards in your graveyard, double the number of +1/+1 counters on those creatures.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature | TgtPrompt$ Select target creature  | CounterType$ P1P1 | CounterNum$ 2 | TargetMin$ 1 | TargetMax$ 2 | DividedAsYouChoose$ 2 | SubAbility$ DBMultiplyCounter
+SVar:DBMultiplyCounter:DB$ MultiplyCounter | Defined$ Targeted | CounterType$ P1P1 |  CheckSVar$ X | SVarCompare$ GE6
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/omnivorous_flytrap.txt:8]: wrong syntax for condition

File: forge-gui/res/cardsfolder/upcoming/patchwork_beastie.txt [hunk 2]
```diff
@@ -2,9 +2,10 @@ Name:Patchwork Beastie
 ManaCost:G
 Types:Artifact Creature Beast
 PT:3/3
-S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't attack or block. | Condition$ Delirium | Delirium — CARDNAME can't attack or block unless there are four or more card types among cards in your graveyard.
+S:Mode$ Continuous | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME can't attack or block. | CheckSVar$ X | SVarCompare$ LT4 | Delirium — CARDNAME can't attack or block unless there are four or more card types among cards in your graveyard.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/patchwork_beastie.txt:5]: why revert?

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/patchwork_beastie.txt:5]: Because it should be unable to attack or block when the delirium is NOT satisfied.

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/patchwork_beastie.txt:5]: haha my bad
but you can use `Count$Delirium.0.1`
and it still needs Description$ :p

**Review by tool4ever — APPROVED** [review]

**Pull request merged**